okhttp/src/main/java/com/squareup/okhttp/CertificatePinner.java

```diff
@@ -123,17 +123,45 @@ private CertificatePinner(Builder builder) {
    * pinned for {@code hostname}. OkHttp calls this after a successful TLS
    * handshake, but before the connection is used.
    *
+   * @return {@code true} if hostname has pinned certificates and check passed,
```

Comment:
Done -> extracted method `findMatchingPins(hostname)`, added several tests for it. I left tests for `check()` with wildcard pattern, because I think that `CertificatePinner` should be well tested.

Thank you for preventing me from making API worse :+1: